### PR TITLE
monitoring: cr example

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -256,27 +269,6 @@ metadata:
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
-  relatedImages:
```
Comment (miguelsorianod) [inline deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml:259]: Is this removal intentional?

File: deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -435,7 +427,7 @@ spec:
                   value: centos/postgresql-10-centos7
                 - name: ZYNC_POSTGRESQL_IMAGE
                   value: centos/postgresql-10-centos7
-                image: quay.io/3scale/3scale-operator:master
```
Comment (miguelsorianod) [inline deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml:438]: And this one?

File: deploy/crds/apps.3scale.net_v1alpha1_apimanager_cr_monitoring.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: apps.3scale.net/v1alpha1
+kind: APIManager
+metadata:
+  name: example-apimanager-s3
```
Comment (miguelsorianod) [inline deploy/crds/apps.3scale.net_v1alpha1_apimanager_cr_monitoring.yaml:4]: Is this example name intentional or should we choose another one?

Comment (eguzki) [reply deploy/crds/apps.3scale.net_v1alpha1_apimanager_cr_monitoring.yaml:4]: other one

File: deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (eguzki) [reply deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml:259]: operator-sdk removed.

unacceptable
